### PR TITLE
Feature/mobile devices

### PR DESCRIPTION
**Pull request by kubatatami**

### CODE REVIEW

File: sdk/src/main/java/io/matchmore/sdk/MatchMoreConfig.kt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package io.matchmore.sdk
+
+import android.content.Context
+
+data class MatchMoreConfig @JvmOverloads constructor(
```
Comment (maciejburda) [inline sdk/src/main/java/io/matchmore/sdk/MatchMoreConfig.kt]: that's something I will incorporate into ios sdk as well.

File: sdk/src/main/java/io/matchmore/sdk/MatchMore.kt [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        if (isConfigured()) throw IllegalStateException("You can not overwrite the configuration.")
+        this.matchMoreConfig = matchMoreConfig
+    }
+
+    @JvmStatic
+    fun isConfigured() = this.matchMoreConfig != null
+}
+
+interface MatchMoreSdk {
+    fun startUsingMainDevice(success: SuccessCallback<MobileDevice>?, error: ErrorCallback?) = startUsingMainDevice(null, success, error)
```
Comment (maciejburda) [inline sdk/src/main/java/io/matchmore/sdk/MatchMore.kt]: is there any smart way to have one callback in kotlin? In swift you can use enum with internal values.

File: sdk/src/main/java/io/matchmore/sdk/api/SubscriptionApi.kt [hunk 3]
```diff
@@ -0,0 +1,62 @@
+package io.matchmore.sdk.api
```
Comment (maciejburda) [inline sdk/src/main/java/io/matchmore/sdk/api/SubscriptionApi.kt]: is there any way to automatically mark generated files? maybe simple header comment?

**Comment by maciejburda** [comment]
Let's try automated Travis tests in github. I will do the same on iOS SDK.

File: sdk/src/main/java/io/matchmore/sdk/api/SubscriptionApi.kt (hunk 3 above)
Comment (kubatatami) [reply sdk/src/main/java/io/matchmore/sdk/api/SubscriptionApi.kt]: Currently, we can't generate kotlin code so all API classes were generated to temp folder and manually(using Android Studio functionality) converted to kotlin. 
Is there any reason to mark that classes?

File: sdk/src/main/java/io/matchmore/sdk/MatchMore.kt (hunk 2 above)
Comment (kubatatami) [reply sdk/src/main/java/io/matchmore/sdk/MatchMore.kt]: As we discussed it's better in kotlin/java to have 2 lambdas.

File: sdk/src/main/java/io/matchmore/sdk/api/SubscriptionApi.kt (hunk 3 above)
Comment (maciejburda) [reply sdk/src/main/java/io/matchmore/sdk/api/SubscriptionApi.kt]: Sounds good. I just want to understand all parts.

**Pull request merged**